src/defaults/components.js

```diff
@@ -205,8 +205,8 @@ const defaults = {
     text: {
       title: 'Cart',
       empty: 'Your cart is empty.',
-      button: 'Checkout',
-      total: 'Subtotal',
+      button: 'CHECKOUT',
+      total: 'Total',
```

Comment:
This should probably still be `Subtotal`, since discounts can be applied after this (in checkout) and this is before shipping, etc.